src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -118,6 +115,37 @@ for further details.
   margin-top: 106px;
 }
 
+.mainAreaWrapper {
+  padding-left: var(--main-area-padding-left);
+  padding-right: var(--main-area-padding-right);
+  padding-top: var(--main-area-padding-top);
+}
+
+.breadcrumbOptOutGroup {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.optInOutSlot {
+  flex-shrink: 0;
+}
+
+/*
+***
+* classes for the version PageAlert
+***
+*/
+
+.versionAlert {
+  padding-left: var(--main-area-padding-left);
+  padding-right: var(--main-area-padding-right);
+}
+
+.versionAlertLink {
+  color: var(--token-color-foreground-primary) !important;
```

Comment:
❓ Curious why we had to use `!important` here?

Reply:
This makes sure that `color` is the same regardless of the state of the link (hover, visited, active, etc.). Would a comment above this line be valuable?

Reply:
That could be helpful, and seems like a reasonable thing to try and do whenever we reach for `!important`! 👍